### ISSUE TITLE
AI.cpp: shipStrength calculation improvement/question

### ISSUE THREAD

**Issue by nylki** [endless-sky/endless-sky#890]
When I was looking at your loop for calculating `shipStrength` in `AI.cpp` I stumbled over this part:
```.js
if(ogov->AttitudeToward(gov) > 0. && oit->Position().Distance(it->Position()) < 2000.)
  strength += it->Cost();
```
I like the idea that a ships strength is internally considered stronger if other allied ships are around. But it also means, that any real costy ship, that has lots and lots of really weak allied shuttles in a radius of 2000 around it, is considered really strong! Am I correct with this observation?
Wouldn't it make more sense to do:

```.js
strength += oit->Cost();
```

**Comment by endless-sky** [comment]
Yes, that's a typo. Fixed in 2711eb2.

**Comment by endless-sky** [comment]
Closing this because it's included in 0.9.0, which is now released.
